Pipfile

```diff
@@ -26,6 +26,7 @@ py-stackexchange = "~=2.2.7"
 "discord.py" = "~=1.3.4"
 lxml = "~=4.5.2"
 bleach = "~=3.1.5"
+html2markdown = "*"
```

Comment:
```suggestion
```
since the trivia command isn't using this, there's no point of including it here